bcgov/moh-prime
# PRIME-2103 Add loading message to user enrolment submission

[Jira Ticket PRIME-2103](https://bcgovmoh.atlassian.net/browse/PRIME-2103)

Using same PR as the changes were made for user enrolment only then a new ticket created to add the loading indicator to all submissions.

### How to test:

Go through a user enrolment. At the end a spinner should show and text will be displayed as well to tell the user that _Wait patiently, submission could take up to a minute_

PR environment may be slow enough to see the spinners, but if you want you can test locally and add a Thread.Sleep in the submission end point.
